go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/grafana/grafana-plugin-sdk-go
 
 go 1.21
 
+// The v0.120.0 is needed for now to be compatible with grafana/thema.
+replace github.com/getkin/kin-openapi => github.com/getkin/kin-openapi v0.120.0
```

Comment:
I am not a fan of this `replace`s. I know it ensures that the specific version is used but it has happened to me that I have updated a dependency without realizing that there was a replace somewhere in the go.mod file. Isn't just enough to put this comment in the related `require` line?